backtesting/strategy.py

```diff
@@ -64,3 +66,33 @@ def stoploss(self, dataframe: DataFrame) -> DataFrame:
         :rtype: DataFrame
         """
         return
+
+    def join_additional_data(self, dataframe, additional, timeframe, timeframe_additional, ffill=True):
```

Comment:
```suggestion
    def join_additional_data(self, dataframe, additional, timeframe_configured, timeframe_additional, ffill=True):
```
Would make it more clear what 'timeframe' is here.